setup.go

```diff
@@ -33,6 +33,7 @@ const (
 	envLoggingURL = "GOLOG_URL" // url that will be processed by sink in the zap
 
 	envLoggingOutput = "GOLOG_OUTPUT" // possible values: stdout|stderr|file combine multiple values with '+'
+	envLoggingLabels = "GOLOG_LOG_LABELS" // comma-separated key-value pairs, i.e. "app,example_app,dc,sjc-1"
```

Comment:
How about `app:example_app,dc=sjc-1`, or `app=example_app dc=sjc-1`, etc. The current syntax will be very easy to get wrong.

Reply:
Is `app=example_app,dc=sjc-1` suitable? You mixed the assignment operator in the first example and I am not sure if that is on purpose.

Reply:
Ah, sorry, that wasn't on purpose. Your solution also works. Really, anything where I can easily read the labels and understand what's going on.